hdfs3/core.py

```diff
@@ -383,11 +383,26 @@ def exists(self, path):
         return out == 0
 
     def chmod(self, path, mode):
-        """ Mode in numerical format (give as octal, if convenient) """
-        imode = mode_numbers[mode]
+        """Change access control of given path
+
+        Exactly what permissions the file will get depends on HDFS
+        configurations.
+
+        Parameters
+        ----------
+        path : string
+            file/directory to change
+
+        mode : integer
+            As with the POSIX standard, each octal digit refers to
+        user-group-all, in that order, with read-write-execute as the
+        bits of each group. E.g.,
+        - 0o777 -> permissive;
+        - 0o711 -> all permission for owner, read-only for others;
+        - 0o100 -> read-only for owner, nothing for others."""
```

Comment:
If we abide by sphinx standards then we get this pretty API page http://hdfs3.readthedocs.org/en/latest/api.html